lib/qa/authorities/linked_data/generic_authority.rb

```diff
@@ -10,7 +10,8 @@ module LinkedData
     # @see Qa::LinkedDataTermsController#show
     # @see Qa::Authorities::LinkedData::Config
     class GenericAuthority < Base
-      attr_reader :auth_config
+      attr_reader :authority_config
+      private :authority_config
```

Comment:
Similar rename for consistency.  Not used outside of this class, so I made it private.